packages/subproviders/src/subproviders/ledger.ts

```diff
@@ -165,27 +140,28 @@ export class LedgerSubprovider extends BaseWalletSubprovider {
         }
     }
     /**
-     * Sign a personal Ethereum signed message. The signing address will be the one
-     * retrieved given a derivationPath and pathIndex set on the subprovider.
+     * Sign a personal Ethereum signed message. The signing account will be the account
+     * associated with the provided address.
```

Comment:
Think it's easy to confuse devs with the address/accounts naming confusion. Let's try to stick to one or the other. Here it might make sense to pass in `account: string`, and always refer to it as `account`. Thoughts?

Reply:
When I think account I picture a publicKey privateKey combination. Where as address is just a string encoded public key. It doesn't help that the JSON RPC documentation also uses both :P 

```
Parameters
account, message

DATA, 20 Bytes - address
DATA, N Bytes - message to sign
```

Reply:
Haha, yeah it doesn't. We need to be better then JSON RPC 😛